README.md

```diff
@@ -101,312 +39,81 @@ terraform {
 
 provider "iterative" {}
 
-resource "iterative_machine" "machine" {
-    repo = "https://github.com/iterative/cml"
-    driver = "github"
-    labels = "tf"
+resource "iterative_task" "example" {
+  name  = "example"
+  cloud = "aws"
```

Comment:
It's `m` by default, and not required. If we want to showcase power features instead of simplicity, I'm fine with adding it.

_Post scriptum: it's `machine` and not `instance_type`_ 😈

Reply:
updated suggestion